Makefile

```diff
@@ -121,22 +109,29 @@ shell: build ## start a shell inside the build env
 	$(DOCKER_RUN_DOCKER) bash
 
 test: build ## run the unit, integration and docker-py tests
-	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary cross test-unit test-integration-cli test-docker-py
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-unit test-integration-cli test-docker-py
 
 test-docker-py: build ## run the docker-py tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-docker-py
 
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh build-integration-test-binary dynbinary test-integration-cli
 
-test-unit: build ## run the unit tests
-	$(DOCKER_RUN_DOCKER) hack/make.sh test-unit
+dobi:
+	curl -LsS -o dobi https://github.com/dnephin/dobi/releases/download/v0.8/dobi-linux
```

Comment:
Definitely +1 on finding a way to make this more forgiving of other platforms. :smile:

Any invocation of `curl` probably ought to include `-f` so that it passes failure forward to Make (and the build doesn't barrel onwards without `dobi` actually downloaded).